src/sql/cmd/connect.py

```diff
@@ -0,0 +1,18 @@
+try:
+    from jupysql_plugin.widgets import ConnectorWidget
+
+    CONNECTORWIDGET = ConnectorWidget()
+except ModuleNotFoundError:
+    ConnectorWidget = None
```

Comment:
Why is `ConnectorWidget` being set to `None`?

Reply:
It's because`jupysql_plugin` is an optional dependency. It is mentioned in the [issue](https://github.com/ploomber/jupysql/issues/632#issue-1765971254).